cicecore/cicedynB/general/ice_init.F90

```diff
@@ -1141,417 +1141,441 @@ fpfunc_andacc = 1      ! fixed point function for Anderson acceleration: 1: g(x)
-               write(nu_diag,1007) ' threshold_hw     = ', threshold_hw, ' max water depth for grounding ice'
+               write(nu_diag,1002) ' k1               = ', k1, ' : free parameter for landfast ice'
+               write(nu_diag,1002) ' k2               = ', k2, ' : free parameter for landfast ice'
+               write(nu_diag,1002) ' alphab           = ', alphab, ' : factor for landfast ice'
+               write(nu_diag,1002) ' threshold_hw     = ', threshold_hw, ' : max water depth for grounding ice'
+            endif
+            write(nu_diag,1002) ' Ktens            = ', Ktens, ' : tensile strength factor'
+
+            if (kdyn == 3) then
+               write(nu_diag,1020) ' maxits_nonlin    = ', maxits_nonlin,' : max nb of iteration for nonlinear solver'
+               write(nu_diag,1030) ' precond          = ', trim(precond),' : preconditioner for fgmres'
+               write(nu_diag,1020) ' dim_fgmres       = ', dim_fgmres,' : size of fgmres Krylov subspace'
+               write(nu_diag,1020) ' dim_pgmres       = ', dim_pgmres,' : size of pgmres Krylov subspace'
+               write(nu_diag,1020) ' maxits_fgmres    = ', maxits_fgmres,' : max nb of iteration for fgmres'
+               write(nu_diag,1020) ' maxits_pgmres    = ', maxits_pgmres,' : max nb of iteration for pgmres'
+               write(nu_diag,1010) ' monitor_nonlin   = ', monitor_nonlin,' : print nonlinear residual norm'
+               write(nu_diag,1010) ' monitor_fgmres   = ', monitor_fgmres,' : print fgmres residual norm'
+               write(nu_diag,1010) ' monitor_pgmres   = ', monitor_pgmres,' : print pgmres residual norm'
+               write(nu_diag,1030) ' ortho_type       = ', trim(ortho_type),' : type of orthogonalization for FGMRES'
+               write(nu_diag,1009) ' reltol_nonlin    = ', reltol_nonlin,' : nonlinear stopping criterion'
+               write(nu_diag,1009) ' reltol_fgmres    = ', reltol_fgmres,' : fgmres stopping criterion'
+               write(nu_diag,1009) ' reltol_pgmres    = ', reltol_pgmres,' : pgmres stopping criterion'
+               write(nu_diag,1030) ' algo_nonlin      = ', trim(algo_nonlin),' : nonlinear algorithm'
+               write(nu_diag,1010) ' use_mean_vrel    = ', use_mean_vrel,' : use mean of previous 2 iterates to compute vrel'
+               if (algo_nonlin == 'anderson') then
+                  write(nu_diag,1020) ' fpfunc_andacc    = ', fpfunc_andacc,' : fixed point function for Anderson acceleration'
+                  write(nu_diag,1020) ' dim_andacc       = ', dim_andacc,' : size of Anderson minimization matrix'
+                  write(nu_diag,1009) ' reltol_andacc    = ', reltol_andacc,' : relative tolerance for Anderson acceleration'
+                  write(nu_diag,1000) ' damping_andacc   = ', damping_andacc,' : damping factor for Anderson acceleration'
+                  write(nu_diag,1020) ' start_andacc     = ', start_andacc,' : acceleration delay factor'
```

Comment:
I know that's what's currently written in **ice_init** and **ice_dyn_vp** , but it's not really a factor, it's really the iteration at which acceleration start... maybe:
> nonlinear iteration at which acceleration starts

would be more accurate

Reply:
This has been updated.